docs/src/pages/landing-page/QuasarComponents.vue

```diff
@@ -34,24 +42,53 @@
 import { defineComponent, ref, computed } from 'vue'
 import { components } from 'src/assets/landing-page/image-links.js'
 
-const FILTER_CHIPS = [ 'Buttons', 'Inputs', 'Loading', 'Media', 'Navigation', 'Panels & Popups', 'Scroll', 'Tables', 'Others' ]
+const OTHER_CATEGORY = 'other'
+
+const FILTER_CHIPS = [
+  { label: 'Buttons', value: 'button' },
+  { label: 'Inputs', value: 'input' },
+  { label: 'Loading', value: 'loading' },
+  { label: 'Media', value: 'media' },
+  { label: 'Navigation', value: 'navigation' },
+  { label: 'Panels & Popups', value: 'panelAndPopup' },
+  { label: 'Scroll', value: 'scroll' },
+  { label: 'Tables', value: 'table' },
+  { label: 'Others', value: OTHER_CATEGORY }
+]
 
 const componentNameToKebabCase = (componentName) => componentName.replaceAll(' ', '-').toLowerCase()
 
 export default defineComponent({
   name: 'Components',
   setup () {
     const search = ref('')
+    const filterCategory = ref('')
+
+    function filterByCategory (categories, filterCategory) {
+      // If a category was not explicitly set, then we consider it to be of category $OTHER_CATEGORY
```

Comment:
I'd require the categories to be explicitly set instead of setting a default when they don't have one
This would also prevent the need for this function at all since the check becomes just `categories.includes(filterCategory)`

Reply:
Okay, as long as all components have a tag, then this function can be eliminated